intelligence_asn.go

```diff
@@ -52,49 +58,49 @@ type IntelligenceASNSubnetResponse struct {
 // IntelligenceASNOverview get overview for an ASN number
 //
 // API Reference: https://api.cloudflare.com/#asn-intelligence-get-asn-overview
-func (api *API) IntelligenceASNOverview(ctx context.Context, params IntelligenceASNOverviewParameters) ([]ASNInfo, error) {
+func (api *API) IntelligenceASNOverview(ctx context.Context, params IntelligenceASNOverviewParameters) (ASNInfo, error) {
```

Comment:
let's update (all) the method signatures to match the [experimental method signatures](https://github.com/cloudflare/cloudflare-go/blob/master/docs/experimental.md#consistent-crud-method-signatures). prior art: https://github.com/cloudflare/cloudflare-go/blob/master/tunnel.go#L218